C++/syntax_test_c.c

```diff
@@ -1028,3 +1029,11 @@ int control_keywords()
   return 123;
   /* <- keyword.control.flow.return */
 }
+
+typedef struct MyStructure {} MyStructure_t;
```

Comment:
I'd try to keep test cases logically grouped. So moving this one before or after the one in line 290 would be suitable.